MetaMask/metamask-mobile
# feat: Add new privacy policy alert toast

## **Description**

Changes:
Adds a new privacy policy banner that will show up after May 6, 2024. [EDIT HB - date set for June 4, 2023]
If the user closes it manually, the privacy policy banner will not show up again.
If the user does not close the toast manually, the privacy policy banner will cease to show up after a day of being shown for the first time.
Changes the Survey banner so that we can display multiple banners at the same time without overlapping each other.

Considerations:
- Make sure this doesn't show up during onboarding
- This toast will show up even after years of the privacy policy being changed. We should set up a deadline for that or eventually revert this PR.
- Check if copy is correct.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to the home page
2. Check if there's a new privacy banner there (date should be edited in the code so that the privacy policy shows up earlier than May 6, 2024)
3. When clicking the link, it should take you to the Privacy Policy website
4. When closing it, it shouldn't show up again
5. If not closing it manually, the toast should not show up again after 1 day of being shown for the first time. This data is saved in localStorage so editing the timestamp there should be enough to test this feature out.

## **Screenshots/Recordings**

https://github.com/MetaMask/metamask-mobile/assets/11148144/ae6e74fc-0a7f-47ce-8744-4a292875f7b3


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
